src/aind_ephys_rig_qc/test.ipynb

```diff
@@ -0,0 +1,83 @@
+{
+ "cells": [
```

Comment:
was this file added on purpose?

Reply:
I removed this test file. It was added by accident.